src/blockdata/script.rs

```diff
@@ -151,6 +151,8 @@ display_from_debug!(Builder);
 /// would help you.
 #[derive(PartialEq, Eq, Debug, Clone)]
 pub enum Error {
+    /// Something did a non-minimal push
+    NonMinimalPush,
```

Comment:
You might add  https://github.com/bitcoin/bips/blob/master/bip-0062.mediawiki#Push_operators or something to help people.

Reply:
Sure, good link